app/PaymentDrivers/StripePaymentDriver.php

```diff
@@ -313,36 +321,38 @@ public function findOrCreateCustomer(): ?\Stripe\Customer
 
     public function refund(Payment $payment, $amount)
     {
-        $this->gateway();
+        $this->init();
 
-        $response = $this->gateway
-            ->refund(['transactionReference' => $payment->transaction_reference, 'amount' => $amount, 'currency' => $payment->client->getCurrencyCode()])
-            ->send();
+        $response = $this->stripe
+            ->refunds
```

Comment:
any reason for the plural refunds instead of refund?

Reply:
Yes! Stripe SDK is written that way. 

https://stripe.com/docs/api/refunds/create